.github/workflows/load-testing.yml

```diff
@@ -0,0 +1,147 @@
+name: Baseline Load Tests
+
+permissions: {}
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "release*"
```

Comment:
How long does the whole test take? Would it block merging before it completes?

I suggest we schedule this every day against the main and active release (N-2) branches.

Reply:
I think at some point we agreed that we have baseline tests that run for every PR and scaled tests that run daily. I suggest we leave it for every PR because then we would be able to know any regressions in performance.

It takes quite some time. But mostly due to cluster setup. We fan optimize this. It takes as much time as the conformance tests.

Reply:
Is it possible to upload artifacts only if we see a degraded performance?

Reply:
It's possible. And makes sense. Okay, I'll push something like this. 👍